prime-router/docs/onboarding-users/migrating-receivers.md

```diff
@@ -0,0 +1,248 @@
+
+When deciding if a transform should go on the receiver or sender. These are a couple of guidelines to keep in mind [Managing Transforms](../standard-operating-procedure/managing-transforms.md)
+
+
+### 6. Testing in Staging
+Once the settings and transforms have been migrated and tested successfully through the UP locally, we can start testing in staging. 
+First have somebody review your changes by either creating your PR or by sharing your changes. 
+Once they have been reviewed push the settings to staging
+One of the ways to push the settings to staging is by running the PRIME command:
+`./prime multiple-settings set --input {path-to-schema} --env staging`
+
+Follow the steps here [Simple Report Test data](#simple-report-test-data) to send a combination of test messages to the STLT.
+The message should get routed to the STLT if their transport has been set-up correctly.  
+Messages routed to the STLT can be viewed by:
+* Logging in as an Admin to ReportStream
+* Search for the STLT organization under Organizations and Click on Set
+* Navigate to https://staging.reportstream.com/daily-data
+* You can view and download all messages routed to the STLT
+* If for some reason you can't view the message routed to the STLT 
+  * Ensure the filters are set-up correctly and that the message you sent has the right patient state
+  * Wait a couple of minutes, it can take a few mins for SimpleReport to release the message
+
+### 7. Move settings to Production
+The process to move setting to production is the same as in staging. 
+However, we won't be able to send test messages to the STLT. 
+We will have to monitor their channel for a couple of weeks and wait until we see data for the STLT being routed through the UP. 
+Once we have data we can notify the STLT to start reviewing the messages.
+
+### Simple Report Test data
+
+A quick and easy way to get test data to send to a STLT is by going into SimpleReport's test environment https://test.simplereport.gov.   
+Access can be requested on the [shared-simple-report-universal-pipeline](https://nava.slack.com/archives/C0411VC78DN) thread.   
+Instructions on how to send a test message can be found on this youtube playlist https://www.youtube.com/playlist?list=PL3U3nqqPGhab0sys3ombZmwOplRYlBOBF. 
+To route the report to a specific STLT either the patient or facility state needs to updated to the STLT's jurisdiction. Keep in mind that if they are not updated the message might get routed to the incorrect STLT.
+The report sent by SimpleReport can be found in the Azure BlobStorage. The UP message will be stored in the `receive/simple_report.fullelr` and the covid pipeline message will be stored in `receive/simple_report.default`. This message can be used locally to test any new sender or receiver transforms.
+To access the blob storage. Microsoft Storage Explorer needs to be installed and login with your CDC SU credentials.
+
```

Comment:
I would add a little blurb here about also making sure to test using the CSV uploader as well as the manual entry as the messages are structured differently.